spec/integration/transaction-manager.spec.js

```diff
@@ -0,0 +1,98 @@
+import _ from 'lodash'
```

Comment:
Same here, we don't need integration test. They takes time and resources to run as they need to call CT platform.

Reply:
See my previous comment https://github.com/commercetools/commercetools-node-variant-reassignment/pull/15#discussion_r154595587